Libplanet/Net/PeerState.cs

```diff
@@ -0,0 +1,40 @@
+using System;
+
+namespace Libplanet.Net
+{
+    /// <summary>
+    /// Describes state of peers stored in routing table.
+    /// </summary>
+    public class PeerState
+    {
+        public PeerState(BoundPeer peer, DateTimeOffset lastUpdated)
```

Comment:
As other properties' setter are `internal`, this constructor might be better to be `internal` in the same way.